packages/core/src/Stores/traders-hub-store.js

```diff
@@ -273,7 +275,10 @@ export default class TradersHubStore extends BaseStore {
         ) {
             if (this.is_eu_user) return ContentFlag.LOW_RISK_CR_EU;
             return ContentFlag.LOW_RISK_CR_NON_EU;
-        } else if (financial_company?.shortcode === 'svg' && gaming_company?.shortcode === 'svg' && this.is_real) {
+        } else if (
+            ((financial_company?.shortcode === 'svg' && gaming_company?.shortcode === 'svg') || restricted_countries) &&
```

Comment:
Isn't it the same condition?

Reply:
The && is the additional check we do for example indonesia. But some countries like Norway & Australia has only one of those conditions.

Reply:
leave a comment and explain the condition for better code readability

Reply:
Done